central/image/datastore/internal/store/dackbox/store_impl.go

```diff
@@ -661,3 +661,54 @@ func (b *storeImpl) GetKeysToIndex(_ context.Context) ([]string, error) {
 	// DackBox handles indexing
 	return nil, nil
 }
+
+func (b *storeImpl) UpdateVulnState(_ context.Context, cve string, images []string, state storage.VulnerabilityState) error {
```

Comment:
The code makes sense, but I don't know DackBox well enough to know for sure that this is correct... Is there a way we can write a unit test for this function, or is that too complicated?

Reply:
Not a huge deal since DackBox is going away soon

Reply:
The unit test resides at `central/vulnerabilityrequest/manager/requestmgr/manager_impl_test.go`. It is only moved from image-cve dir to here, that's it.